From 9d86c1665159b325fe5f8936eb521f9f283ad668 Mon Sep 17 00:00:00 2001
From: Michael Pankov <constantius9@gmail.com>
Date: Wed, 29 Oct 2014 17:16:02 +0300
Subject: [PATCH 1/2] Change commands for a

---
 proto/cmd/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/proto/cmd/Makefile b/proto/cmd/Makefile
index 0819c9d..a445aa8 100644
--- a/proto/cmd/Makefile
+++ b/proto/cmd/Makefile
@@ -3,7 +3,7 @@ RES = build/res/b
 all: $(RES)
 
 build/aux/build/res/a.cmd.new: Makefile | $(DIRECTORY)
-> echo 'echo A; echo I am a\nWell aaaa azaza > $(call GET_TARGET_PATH,$@)' > $@
+> echo 'echo A; echo I am a > $(call GET_TARGET_PATH,$@)' > $@
 
 build/aux/build/res/b.cmd.new: build/aux/build/res/a.did_update Makefile | $(DIRECTORY)
 > echo 'echo B; cat $< > $(call GET_TARGET_PATH,$@);
-- 
1.9.1

