From 5aa04f6801aefcbb2751e4bd1217c585e440e83b Mon Sep 17 00:00:00 2001
From: Michael Pankov <constantius9@gmail.com>
Date: Wed, 29 Oct 2014 20:20:51 +0300
Subject: [PATCH] Change commands for b

---
 proto/cmd/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/proto/cmd/Makefile b/proto/cmd/Makefile
index 1f7ab7a..ab9ae13 100644
--- a/proto/cmd/Makefile
+++ b/proto/cmd/Makefile
@@ -9,7 +9,7 @@ build/aux/build/res/a.cmd.new: Makefile | $(DIRECTORY)
 
 build/aux/build/res/b.cmd.new: build/aux/build/res/a.did_update Makefile | $(DIRECTORY)
 > echo 'echo B; cat $< > $(call GET_TARGET_PATH,$@);
-> echo And I am b buzzz >> $(call GET_TARGET_PATH,$@)' > $@
+> echo And I am b buzzz bbbb >> $(call GET_TARGET_PATH,$@)' > $@
 
 build/res/%: \
   build/aux/build/res/%.cmd.did_update \
-- 
1.9.1

